telegram/inline/inlinekeyboardbutton.py

```diff
@@ -27,7 +27,8 @@ class InlineKeyboardButton(TelegramObject):
     This object represents one button of an inline keyboard.
 
     Note:
-        You must use exactly one of the optional fields.
+        You must use exactly one of the optional fields. Mind that :attr:`callback_game` is not
```

Comment:
This isn't true though. callback_game the attribute is implemented, but it just accepts a games short name I think, otherwise there would be no way to launch games from custom keyboards.